packages/cli/src/commands/server/debugger-ui/index.html

```diff
@@ -185,27 +181,8 @@
       } else if (object.method === '$disconnected') {
         shutdownJSRuntime();
         Page.setState({status: {type: 'disconnected'}});
-      } else if (object.method === 'executeApplicationScript') {
```

Comment:
is it safe to remove this?

Reply:
Yes, it will simply fall through and call `postMessage` below so it's going to do the same as before, just with less code ;)